### ISSUE TITLE
[Effect Suggestion] My Super Cool Effect Idea

### ISSUE THREAD

**Issue by HarleyQuinn36** [gta-chaos-mod/ChaosModV#1631]
Name of effect
Force play a replay mission

Duration if timed effect (long or short timed?)

Effect Description
similar to force to random mission but this force you to replay mission you already done

**Comment by RosyMiranto18** [comment]
I think this idea may be an interesting one. However, the pool of missions that can get replayed also includes the chance of a Stranger of Freaks even when the odds is really really small

**Comment by HarleyQuinn36** [comment]
> I think this idea may be an interesting one. However, the pool of missions that can get replayed also includes the chance of a Stranger of Freaks even when the odds is really really small

That is true, was referring to all replay able stuff just mentioned them all as mission
